Add NovelAI Image Generator Enhancements doc

esbe1175 opened the pull request:
Per request, hope it's up to standards.

esbe1175 committed:
Create novelai-image-generator-enhancements.md

esbe1175 committed:
Update novelai-image-generator-enhancements.md

TapwaveZodiac commented:
A good start! I would put installation earlier in the document, just before the in-detail feature explanations. There're a few capitalization/punctuation omissions to fix as well, if you want to make it look nice.

esbe1175 committed:
Update novelai-image-generator-enhancements.md

Moved Installation instructions near the top (TL;DR). Capitalization, typos, wording.

esbe1175 commented:
Updated the file, I do think it's a good idea to put install instructions earlier for clarity. Also reworded some things and corrected grammatical mistakes, capitalization etc.
